src/resources/health-hygiene/PDF/resources/view.html

```diff
@@ -14,21 +14,21 @@ <h3>Do Food Safely</h3>
     <div class="qg-card__footer">
       <a
         class="anchor-margin"
-        href="https://dofoodsafely.health.vic.gov.au/index.php/en/"
+        href="http://dofoodsafely.health.vic.gov.au/index.php/en/"
```

Comment:
is this website alive?

Reply:
We did comment wondering how they can do food safely but not website security.

Reply:
just trying to go to it gives me
504 Gateway Time-out
The server didn't respond in time.

Reply:
The server is meant to be live, it was on Thursday, and is still linked to by the vic health website.
Eg: 
https://www2.health.vic.gov.au/public-health/food-safety/food-safety-information-for-community-groups/food-safety-training-skills-knowledge/do-food-safely

Reply:
I emailed the vic.gov that was linked on a cached copy of dofoodsafely that it did not have ssl and may have been hacked/malware/crypto miner infected as its offline.